src/main/java/seedu/address/model/country/CountryManager.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.country;
+
+import java.util.HashMap;
+import java.util.Locale;
+
+/**
+ * A high-level class responsible for mapping ISO3166 2-letter country codes to countries.
+ */
+public class CountryManager {
+
+    //TODO: Should include checking for 3-letter Country Code?
+    private final HashMap<String, Country> countryCodeMap;
```

Comment:
Constant names must be all uppercase using underscore to separate words.

Is there a need for this Map? From what I see, I think we can just keep the ISOCountry String array, then check if the entered String is inside, before initializing a new Locale with that String. This way, we won't have to keep so many country objects, since most of the countries likely won't be used.

Reply:
I'm not quite sure what you mean. If the user enters a valid country code, then just create a new Country object on demand?

Reply:
Ah ok nvm I think can just stick with this for now. I was thinking of something else to prevent the unnecessary creation of objects, but this HashMap is still necessary for it, my bad.

Btw can change the HashMap to Map, so that we standardize

Reply:
Yup changed